apps/data-types.md

```diff
@@ -237,12 +239,17 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | billingAddress | [address](page:apps-data-types#address) | The billing address for the order.  |
 | shippingAddress | [address](page:apps-data-types#address) | The shipping address for the order.  |
 | invoicedOn | string | The date/time the order was invoiced. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
+| partialyPaidOn | string | The date/time a part of the order was paid. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
 | deliveredOn | string | The date/time the order was delivered. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
+| partialyInvoicedOn | string | The date/time a part of the order was invoiced. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
```

Comment:
Well, as already mentioned yesterday, I was not up-to-date about some other changes on the API, so I decided to include it and change the type, once it's merged. I don't want to risk another "out-of-date"-ness :-)